tools/lang_demo.py

```
@@ -0,0 +1,388 @@
+    exclude = {m.strip() for m in args.exclude.split(';') if m.strip()}
+    langs = [s.strip() for s in args.langs.split(',') if s.strip()]
+
+    pk = os.path.join(args.qmk, 'keyboards', 'handwired', 'polykybd')
+    keyboard_json = os.path.join(pk, 'split72', 'keyboard.json')
+    keymap_c = os.path.join(pk, 'split72', 'keymaps', 'default', 'keymap.c')
+
+    matrices = parse_layout_matrix(keyboard_json)
+    kcs = parse_base_layer_keycodes(keymap_c)
+    if len(matrices) != len(kcs):
+        raise SystemExit(f"layout/keymap length mismatch: {len(matrices)} vs {len(kcs)}")
+    matrix_kc = dict(zip(matrices, kcs))
```

Ruff (0.15.18)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)